components/date_picker/theme.css

```diff
@@ -207,40 +207,44 @@
   }
 }
 
-.slideRightLeave {
+.slideRightExit {
   opacity: 1;
   transform: translateX(0);
 
-  &.slideRightLeaveActive {
+  &.slideRightExitActive {
     opacity: 0;
     transform: translateX(-100%);
   }
 }
 
 .slideLeftEnter,
-.slideLeftLeave {
+.slideLeftExit {
   position: absolute;
-  transition-duration: 0.35s;
+}
+
+.slideLeftEnterActive,
+.slideLeftExitActive {
+  transition-duration: 350ms;
   transition-property: transform, opacity;
   transition-timing-function: ease-in-out;
 }
 
 .slideLeftEnter {
   opacity: 0;
-  transform: translate3d(-100%, 0, 0);
+  transform: translateX(-100%);
 
   &.slideLeftEnterActive {
     opacity: 1;
-    transform: translate3d(0, 0, 0);
+    transform: translateX(0);
```

Comment:
Why aren't we using `translate3d` anymore? It's supposed to be better since it relies on the GPU afaik